magento/magento2
# Do not disable maintenance mode after running a backup.

### Description

Keep the initial maintenance mode state before running the backup.
Only disable maintenance mode if it was not setted before.

Github issue: https://github.com/magento/magento2/issues/7951


### Fixed Issues (if relevant)

1. magento/magento2#7951: Running setup:backup with maintenance mode already on, disables after completion.

### Manual testing scenarios

1. run > bin\magento maintenance:enable
2. run > bin\magento setup:backup --db
3. when backup is complete, you don't see:
"[SUCCESS]: DB backup completed successfully.
Disabling maintenance mode"

### Contribution checklist
